### ISSUE TITLE
About Test

### ISSUE THREAD

**Issue by nyyyyyyyyn** [HKUST-Aerial-Robotics/SIMPL#32]
Thank you for your awesome contribution!
Is it possible to release test related code?
Looking forward to your reply!

**Comment by XinGP** [comment]
You can refer to the test file in my code repository, the framework is based on Simpl. https://github.com/XinGP/DGFNet

**Comment by Family-Liao** [comment]
Have you implemented the test.py to generate the h5 for Simpl @nyyyyyyyyn

**Comment by nyyyyyyyyn** [comment]
thanks！



---Original---
From: ***@***.***&gt;
Date: Tue, Apr 8, 2025 22:36 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [HKUST-Aerial-Robotics/SIMPL] About Test (Issue #32)


   
Have you implemented the test.py to generate the h5 for Simpl @nyyyyyyyyn

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: ***@***.***&gt;
  Family-Liao left a comment (HKUST-Aerial-Robotics/SIMPL#32)
 
Have you implemented the test.py to generate the h5 for Simpl @nyyyyyyyyn

**Comment by nyyyyyyyyn** [comment]
thanks！



---Original---
From: ***@***.***&gt;
Date: Mon, Mar 17, 2025 21:02 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [HKUST-Aerial-Robotics/SIMPL] About Test (Issue #32)


   
You can refer to the test file in my code repository, the framework is based on Simpl. https://github.com/XinGP/DGFNet

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***&gt;
  XinGP left a comment (HKUST-Aerial-Robotics/SIMPL#32)
 
You can refer to the test file in my code repository, the framework is based on Simpl. https://github.com/XinGP/DGFNet

**Comment by nyyyyyyyyn** [comment]
yes,I have solved it.I think the reason for the previous mistake was that I didn't perform a rotation matrix operation on the coordinates.



---Original---
From: ***@***.***&gt;
Date: Tue, Apr 8, 2025 22:36 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [HKUST-Aerial-Robotics/SIMPL] About Test (Issue #32)


   
Have you implemented the test.py to generate the h5 for Simpl @nyyyyyyyyn

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: ***@***.***&gt;
  Family-Liao left a comment (HKUST-Aerial-Robotics/SIMPL#32)
 
Have you implemented the test.py to generate the h5 for Simpl @nyyyyyyyyn

**Comment by Family-Liao** [comment]
Could you please send the right test.py file to me for testing the result of simpl. Thanks a lot! My e-mail address is 1311562375@qq.com .Wish you all best! @nyyyyyyyyn

**Comment by Family-Liao** [comment]
> yes,I have solved it.I think the reason for the previous mistake was that I didn't perform a rotation matrix operation on the coordinates.
> […](#)

bro, how did you locate the mistake which is caused by a rotation transform,and is it possible to send the right test related code to me.My e-mail address is 1311562375@qq.com! Looking forward to your reply! Sincere! @nyyyyyyyyn

**Comment by Family-Liao** [comment]
could you please tell me where you take the matrix rotation operation to get the right result? @nyyyyyyyyn

**Comment by Family-Liao** [comment]
please give me a hand bro, i cant fix the bug @nyyyyyyyyn
